Readme.md

```diff
@@ -0,0 +1,79 @@
+# SellEverywhere
+
+## Overview
+We want to create a hardware shop using popular technologies in a friendly team
+(MKdir). Examples are any store selling goods. Current idea is to implement an
+admin panel and user panel. The basic interface of the store was designed in [Figma](https://www.figma.com/file/Re4tXW7aNCWOzed7wI1GKe/dirStore).
+
+### Basic flow
+![Basic flow](./docs/basic_flow.png)
+
+### The application must have the following functionality from the administrator perspective:
+
+* authorization - jwt;
+* authentication;
+* creation of a new product (fill price, description, category, category characteristics, name, manufacturer, photo);
+* fill characteristics of the product by chosen category (dynamic characteristics)
+* manage existing products/deliverers (read/edit/delete);
+* promotion ability helps admin to change product price and see history of changes and add product to the list of special propositions
```

Comment:
event sourcing for prices?

Reply:
@nosovk, don`t understand your opinion. Why administrator should do this?

Reply:
> see history of changes
it means that we need to store versions...

Reply:
@nosovk, I think we can keep JSON with prices changes.